### PR TITLE
Update /support for landscape-self-service

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Some small copy updates and a new "Systems management tool: Landscape" section.

## QA
- Pull code and run `./run`
- Go to http://localhost:8001/support
- Check that the copy matches the copy doc

Copy doc https://docs.google.com/document/d/1liY2ulrI3JiICSJquTn9cH0Hu-R6H3ufmEmYhive-QI/edit?pli=1#

### CODE REVIEW

**Comment by anthonydillon** [comment]
[Update /support for landscape-self-service](https://trello.com/c/uQzWt9PC/2986-update-support-for-landscape-self-service)

File: templates/shared/_systems_management_tool.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<section class="row row-grey row-system-management">
+    <div class="strip-inner-wrapper">
+        <div class="seven-col">
+            <h2>Systems management tool: Landscape</h2>
+            <p>
+                Reduce your team's efforts on day-to-day management with
```
Comment (grahambancroft) [inline templates/shared/_systems_management_tool.html]: Should the apostrophe be encoded?

File: templates/support/index.html [hunk 2]
```diff
@@ -37,30 +37,61 @@ <h1 class="u-full-width">Ubuntu Advantage commercial support</h1>
...
+                    <a href="https://www.ubuntu.com/legal/ubuntu-advantage/assurance">
+                        Ubuntu Assurance Programme
+                    </a> that provides indemnification against claims of
+                    intellectual property infringement.
+                </p>
             </li>
             <li class="grid-list__item four-col last-col">
-                <h3>Access to our world-class technical team</h3>
-                <p>Always on access to the world’s leading Ubuntu desktop, server and cloud experts, all actively participating in open source projects. </p>
+                <h3>Access to our world-class technical team &amp; library</h3>
```
Comment (grahambancroft) [inline templates/support/index.html]: Should be ‘and’ not ampersand, noted in the doc

File: templates/shared/_systems_management_tool.html (hunk 1 above)
Comment (anthonydillon) [inline templates/shared/_systems_management_tool.html]: done

File: templates/support/index.html (hunk 2 above)
Comment (anthonydillon) [inline templates/support/index.html]: Done, thanks

**Comment by grahambancroft** [comment]
Lgtm 👍

**Pull request merged**